web/src/main/java/ua/com/vertex/beans/CourseUserDto.java

```diff
@@ -0,0 +1,104 @@
+package ua.com.vertex.beans;
+
+import java.util.Objects;
+
+public class CourseUserDto {
```

Comment:
The difference is like between basketball and hockey. These are completely different classes. Dto combines several fields from Course, several fields from User and a couple of fields that are present in neither of those classes. The other question is that it may disappear after refactoring in Course_Users table in the database will have been finished.